components/StyledNavContainer.js

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+export const StyledNavContainer = styled.nav`
+  position: fixed;
+  top: 60px;
+  left: 0;
+  width: 250px;
+  height: 100%;
+  background: #f2f2f2;
+  z-index: 8888;
+  transition: transform 0.3s ease-in-out;
+  transform: ${({ isOpen }) =>
+    isOpen ? "translateX(0%)" : "translateX(-100%)"};
+`;
+
+export const StyledTransparentContainer = styled.nav`
```

Comment:
Hi Ralf, thank you for your comments
At the moment i don't have a single approach how to implement this. For my purposes, it works at the moment and I would like to keep it that way, because I'm already sitting on this UserStory for 3 days and still have a lot of work ahead of me....

Reply:
@actyralf 
Hi Ralf, thank you for your comments
At the moment i don't have a single approach how to implement this. For my purposes, it works at the moment and I would like to keep it that way, because I'm already sitting on this UserStory for 3 days and still have a lot of work ahead of me....

Reply:
I think Ralf meant that you can use props do adapt your styled components. [You can check the docs (for later).](https://neuefische-students.slack.com/archives/C04RE009UTS/p1683786453421679)